src/solvers/euler2_solver.C

```diff
@@ -257,6 +261,232 @@ bool Euler2Solver::_general_residual (bool request_jacobian,
   return jacobian_computed;
 }
 
+void Euler2Solver::integrate_qoi_timestep()
+{
+  // We are using the trapezoidal rule to integrate each timestep
+  // (f(t_j) + f(t_j+1))/2 (t_j+1 - t_j)
```

Comment:
Looks like we're integrating consistently with theta below.  Update this comment accordingly?

Reply:
Done.